src/main/resources/english.yml

```diff
@@ -1,5 +1,5 @@
 name: SiegeWar
-version: 0.11
+version: 0.12
 language: english

```

Comment:
Doesn't need to be bumped, it was already bumped since 0.2.1

Reply:
Fixed